drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -283,6 +283,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     if (!window.gettext) { window.gettext = function gettext_stub(string) { return string; }; }
 
     var $element = $(element);
+    element = $element[0]; // <- workaround Studio bug until https://github.com/edx/edx-platform/pull/11433 is merged
```

Comment:
I think it's fine to leave it in for compatibility reasons, but in that case, it'd be nice to have the comment updated to reflect exactly what needs to happen before it can be removed. I might be helpful to write the new comment as a `// TODO: ` as well to make it easier to find later and let people know that it's an action that should be taken later.